spring-integration-core/src/test/java/org/springframework/integration/handler/ServiceActivatorDefaultFrameworkMethodTests.java

```diff
@@ -221,4 +277,19 @@ public String processMessage(Message<?> message) {
 		}
 	}
 
+	private static class AsyncService {
```

Comment:
Ah! This one passes the check because it doesn't have an explicit `public` ctor like those :smile: